ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -2031,6 +2032,24 @@ def test033_Duplicate_main_order_and_cahange_materiel_type(self):
         self.assertEqual(suborder_data['Test Units'], test_unit)
         self.assertEqual(suborder_data['Test Plans'], selected_test_plan['testPlanName'])
 
+    def test035_archived_test_unit_shoudnt_display_in_the_order_drop_down_list(self):
+        """
+        Orders: Archived Test unit: Archive Approach: Archived test units shouldn't appear in orders in the drop down list
+        LIMS-3710
+        :return:
+        """
+        re, payload = self.test_unit_api.create_qualitative_testunit()
+        archived_testunit = self.test_unit_api.archive_testunits(ids=[str(re['testUnit']['testUnitId'])])
+        self.order_page.get_orders_page()
+        self.base_selenium.click(element='orders:new_order')
+        self.order_page.set_new_order()
+        self.order_page.sleep_small()
+        self.order_page.set_material_type_of_first_suborder(material_type='r', sub_order_index=0)
```

Comment:
You should use the material type that has been used in creating the test unit. Right?

Reply:
@0xIslamTaha No,  the test unit creation using the api, it set the material type by default to (All ) but In the test plans and the orders we don't have all material type so this mean If I select any material type for sure this test unit appear because I created it with ( All)